eth/gasprice/gasprice_test.go

```diff
@@ -198,6 +198,7 @@ func TestSuggestTipCap(t *testing.T) {
 
 		// The gas price sampled is: 32G, 31G, 30G, 29G, 28G, 27G
 		got, err := oracle.SuggestTipCap(context.Background())
+		backend.chain.Stop()
```

Comment:
It feels weird to call chain stop here. Maybe we should define a teardown function for the backend and call defer backend.teardown after initializing backend?